src/cli/generator-html.ts

```diff
@@ -241,9 +241,9 @@ function useComponentFunc(element: UseComponent, ctx: GeneratorContext) {
     const componentNode = new CompositeGeneratorNode();
     const refContent = element.component.ref?.content as BodyElement[];
     const refParameters = element.component.ref?.parameters as Parameter[];
-    const argumentList = refParameters.map(function (refEl: Parameter, index: integer) {
-        return ({ name: refEl.name, type: refEl.type, value: generateExpression(element.arguments[index], ctx) });
-    });
+    const argumentList = refParameters.map((refEl: Parameter, index: integer) =>
+        ({ name: refEl.name, type: refEl.type, value: generateExpression(element.arguments[index], ctx) })
+    );
```

Comment:
lambdas are the more _modern_ and _concise_ way to go when passing _behavior_ to a function (that is called _higher order function_).